config/Migrations/20171019083500_fix_cloumn_length.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+use Phinx\Migration\AbstractMigration;
+
+class ImprovementsForMysql extends AbstractMigration {
+
+	/**
+	 * Change Method.
+	 *
+	 * Write your reversible migrations using this method.
+	 *
+	 * More information on writing migrations is available here:
+	 * http://docs.phinx.org/en/latest/migrations.html#the-abstractmigration-class
+	 *
+	 * @return void
+	 */
+	public function change() {
+		$table = $this->table('queue_processes');
+
+		$table->changeColumn('pid', 'string', [
+		   'length' => 40,
```

Comment:
Why is this necessary?

Reply:
`pid` is set by https://github.com/dereuromark/cakephp-queue/blob/9759da22cc678fc6fc0516bc31774c0b662e298e/src/Model/Table/QueuedJobsTable.php#L479 PHP Doc of sha1 says

>If the optional raw_output is set to TRUE, then the sha1 digest is instead returned in raw binary format with a length of 20, otherwise the returned value is a 40-character hexadecimal number. 

Due to this we need 40 digits